src/map.cpp

```diff
@@ -8440,6 +8425,21 @@ tripoint_range map::points_in_radius( const tripoint &center, size_t radius, siz
     return tripoint_range( tripoint( minx, miny, minz ), tripoint( maxx, maxy, maxz ) );
 }
 
+tripoint_range map::points_on_zlevel( const int z ) const
+{
+    if( z < -OVERMAP_DEPTH || z > OVERMAP_HEIGHT ) {
+        // @todo need a default constructor that creates an empty range.
```

Comment:
#28832
```suggestion
        // TODO: need a default constructor that creates an empty range.
```

Reply:
I use @todo because that is recognized by doxygen (see http://dev.narc.ro/cataclysm/doxygen/todo.html).

But apparently it must be doxygen comments, not normal comments.

Anyway: is there an official required style for this?

Reply:
Leveraging oxygen is a good reason to do it that way. 
I'm not too bothered, just do a case insensitive search for "todo".